rfcs/text/0000-contributed-pipelines.md

```diff
@@ -0,0 +1,161 @@
+
+## Author(s)
+ [Kylee Degatano](mailto:kdegatano@broadinstitute.org)
+ 
+ [Ambrose Carr](mailto:acarr@chanzuckerberg.com)
+ 
+ In partnership with Kathleen Tibbetts, Tim Tickle, Clare Bernard, Marcus Kinsella, and the DCP Data Pipelines team.
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+As the DCP accepts data from assays, it takes on responsibility for eventually processing that data and returning analysis products to our contributors. The core mission of the HCA DCP is to process and make available the diverse data types comprising the reference atlas. 
+
+This is a complex task best accomplished by learning from the scientific community. By rapidly inducting existing pipelines into the DCP, and then improving them based on user feedback and demand, we can quickly build capacity to process diverse data types. By focusing on adding breadth across data types, we provide a platform to advance computational and assay diversity in support of the HCA, improving its quality and accelerating its construction.
+ 
+This RFC discusses (1) technical and scientific standards to determine when pipelines contributed by community members are ready for inclusion in the DCP (2), a potential contribution process, (3) the prioritization of such pipelines, and (4) off-boarding pipelines when they lose value or cease to fulfill standard requirements.
+
+## User Stories
+- As a user of the DCP (both user archetypes, but researcher with a pipette especially), I am looking for all raw data in the HCA DCP to be analyzed. I trust the scientific community to write high quality pipelines. 
+- As a computational biologist or methodologist, I have a pipeline that I would like to see leveraged in the HCA DCP.
+- As the HCA DCP, I want to develop pipelines iteratively based on user feedback. To accomplish this, I need new pipelines to be leveraged in the DCP as quickly as possible. 
+
+## Definitions
+**Assay:** A biological and technological process that transforms a biological specimen into interpretable raw data, ideally of a known and standardized format. Generates output which must adhere to a specific raw data format.
+
+**Pipeline:** A collection of one or more functional tasks that operate on input data and, from that, transform the input data, or derive features often used to interpret the input data. In a high throughput setting, these tasks are often automated to be performed in a batch setting.
+
+## Detailed Design
+### Criteria for Consideration
+#### Technical standards for consideration
+The DCP commits to processing included assay types with high quality pipelines that can operate at scale. Thus, there are some basic technical standards that a pipeline must deliver to be eligible for the DCP. A pipeline must: 
```

Comment:
@barkasn made a general comment about this as well but I'd like to understand better what "high quality" means. There are mentions of testing in the bullet points below but are we requiring a minimum test coverage for example? How many datasets should be available for use? How are "acceptable ranges and a method of validation" determined?